src/devices/m_bus.c

```diff
@@ -210,30 +196,265 @@ static char* oms_hum_el[4][4] = {
 {"Error 31","Error 32","Error 33","Error 34",}
 };
 
+static char* history_hours[4] = {
```

Comment:
type should be `char *x` instead of `char* x`.
indent these lists by 8 spaces.

Reply:
@zuckschwerdt  are you sure? I see the "static char* oms_hum" uses c++ asterix position (like type is char*, not *osm_hum). Indentation the same. I can change it every where to have it consistent. If needed :)

Reply:
Yes, but only fix your code for now to keep the diff readable. You can also check with clang-format for a rough guide.